app/views/hub/efile_errors/index.html.erb

```diff
@@ -21,16 +23,16 @@
         <th scope="col" width="500" class="index-table__header">
```

Comment:
another thing i wanted to try on this table was to see if it was as easy as adding a class to make the table header sticky... the check boxes lose context once you scroll past the top of the page and now weve got lots of error messages.